sdk/logs/src/test/java/io/opentelemetry/sdk/logs/AnyValueBodyTest.java

```diff
@@ -95,40 +102,48 @@ void anyValueBody() {
         .hasSize(1)
         .satisfiesExactly(
             logRecordData -> {
-              // TODO (jack-berg): add assertion when ANY_VALUE is added to Body.Type
-              // assertThat(logRecordData.getBody().getType()).isEqualTo(Body.Type.ANY_VALUE);
-              assertThat(logRecordData.getBody().asString())
-                  .isEqualTo(
-                      "["
-                          + "str_key=value, "
-                          + "bool_key=true, "
-                          + "long_key=1, "
-                          + "double_key=1.1, "
-                          + "bytes_key=Ynl0ZXM=, "
-                          + "arr_key=[entry1, 2, 3.3], "
-                          + "key_value_list_key=[child_str_key1=child_value1, child_str_key2=child_value2]"
-                          + "]");
-              assertThat(((AnyValueBody) logRecordData.getBody()).asAnyValue())
-                  .isEqualTo(
-                      AnyValue.of(
-                          KeyAnyValue.of("str_key", AnyValue.of("value")),
-                          KeyAnyValue.of("bool_key", AnyValue.of(true)),
-                          KeyAnyValue.of("long_key", AnyValue.of(1L)),
-                          KeyAnyValue.of("double_key", AnyValue.of(1.1)),
-                          KeyAnyValue.of(
-                              "bytes_key", AnyValue.of("bytes".getBytes(StandardCharsets.UTF_8))),
-                          KeyAnyValue.of(
-                              "arr_key",
-                              AnyValue.of(AnyValue.of("entry1"), AnyValue.of(2), AnyValue.of(3.3))),
-                          KeyAnyValue.of(
-                              "key_value_list_key",
-                              AnyValue.of(
-                                  new LinkedHashMap<String, AnyValue<?>>() {
-                                    {
-                                      put("child_str_key1", AnyValue.of("child_value1"));
-                                      put("child_str_key2", AnyValue.of("child_value2"));
-                                    }
-                                  }))));
+              assertThat(logRecordData.getAnyValueBody())
```

Comment:
We could add a TODO here about using the fluent asserts once #6509 is ready. Just a thought.